### PR TITLE
Pdi 1542

### PR DESCRIPTION
**Pull request by erikostien-pingidentity**

### CODE REVIEW

File: internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		)
+
+		switch {
+		case app.ApplicationOIDC != nil:
+			appId, appIdOk = app.ApplicationOIDC.GetIdOk()
+			appName, appNameOk = app.ApplicationOIDC.GetNameOk()
+		case app.ApplicationSAML != nil:
+			appId, appIdOk = app.ApplicationSAML.GetIdOk()
+			appName, appNameOk = app.ApplicationSAML.GetNameOk()
+		default:
```
Comment (wesleymccollam) [inline internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go:59]: looks like you're missing a case for app.ApplicationExternalLink here

**Review by wesleymccollam — CHANGES_REQUESTED** [review]
just one potential change needed here

File: internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go (hunk 1 above)
Comment (wesleymccollam) [reply internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go:59]: See the pingone provider resource logic
https://github.com/pingidentity/terraform-provider-pingone/blob/e169edbe77031bc1e72309d77241a9776cc1566c/internal/service/sso/resource_application.go#L734

Comment (erikostien-pingidentity) [reply internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go:64]: Added WSFED

**Review by wesleymccollam — APPROVED** [review]

**Pull request merged**